geotrellis-sentinelhub/src/main/scala/org/openeo/geotrellissentinelhub/PyramidFactory.scala

```diff
@@ -334,10 +334,31 @@ class PyramidFactory(collectionId: String, datasetId: String, catalogApi: Catalo
 
             tracker.addInputProducts(collectionId, features.keys.toList.asJava)
 
-            val featureIntersections = for {
-              feature <- sc.parallelize(features.values.toSeq, math.max(1, features.size / 10))
-              reprojectedFeature = feature.reproject(LatLng, boundingBox.crs)
-            } yield Feature(reprojectedFeature intersection multiPolygon, reprojectedFeature.data.toLocalDate.atStartOfDay(UTC))
+            val featuresRDD = sc.parallelize(features.values.toSeq, math.max(1, features.size / 10))
```

Comment:
A good opportunity to get rid of `math.max`: a Scala Int already has a `max` method so replace this with: `1 max (features.size / 10)`

Reply:
fixed